drop_dimension.json

```diff
@@ -0,0 +1,44 @@
+{
+    "id": "drop_dimension",
+    "summary": "Removes a dimension",
+    "description": "Drops a dimension from the data cube.\n\nDropping a dimension only works on dimensions with a single dimension value left, otherwise the process fails with a `TooManyDimensionValues` error. Dimension values can be reduced to a single value with a filter such as ``filter_bands()`` or the ``reduce()`` process. If a dimension with the specified name does not exist, the process fails with a `DimensionExists` error.",
```

Comment:
> process fails with a `DimensionExists` error

should be `DimensionNotAvailable` as specified lower?

Reply:
Thanks, that was too much "copy&paste" from add_dimension.